app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A simple, Unicode-aware word processor that runs on the console"
+DESCRIPTION="WordGrinder is designed to get out of your way and let you write; \
+it does very little, but what it does it does it well. It supports basic paragraph \
+styles, basic character styles, basic screen markup, a menu interface that means you \
+don't have to remember complex key sequences, HTML import and export, and some other \
+useful features."
+HOMEPAGE="http://cowlark.com/wordgrinder/"
+COPYRIGHT="2007-2017 David Given"
+LICENSE="MIT"
+REVISION="1"
+gitSrcRevision="3b07ccffc3043ef03a589e7ff3ab22240e5dcdda"
+SOURCE_URI="https://github.com/davidgiven/wordgrinder/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="713543928e7b5dd4677e7edcf5351f75ce63bf62db316cbf761868fd86cc79c5"
+SOURCE_DIR="wordgrinder-$gitSrcRevision"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	wordgrinder = $portVersion
+	app:WordGrinder = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:ninja
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
```

Comment:
This might help, but I don't know if it will be enough.
```diff
-	make OBJ_DIR=objects
+	make PREFIX=$prefix BINDIR=$binDir MANDIR=$manDir
```
I think you will also need to add one of the versions of Lua: lib:liblua5 to REQUIRES and devel:liblua5 to BUILD_REQUIRES. (I'm not sure which version of Lua should be used.)